apps/openchallenges/organization-service/src/test/java/org/sagebionetworks/openchallenges/organization/service/exception/GlobalExceptionHandlerTest.java

```diff
@@ -0,0 +1,54 @@
+package org.sagebionetworks.openchallenges.organization.service.exception;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Locale;
+import org.junit.jupiter.api.Test;
+import org.sagebionetworks.openchallenges.organization.service.model.dto.BasicErrorDto;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+public class GlobalExceptionHandlerTest {
+
+  @Test
+  public void HandleException_ShouldReturnStatus_WhenExceptionAndLocalePassed() {
```

Comment:
Should use the exact name of the resource: `GlobalExceptionHandler`.

```suggestion
  public void GlobalExceptionHandler_ShouldReturnStatus_WhenExceptionAndLocalePassed() {
```